src/react/components/pages/editorPage/tableView.tsx

```diff
@@ -50,7 +50,8 @@ export const TableView: React.FunctionComponent<ITableViewProps> = (props) => {
             <Modal
                 titleAriaId={"Table view"}
                 isOpen={props.tableToView !== null}
-                isModeless={true}
+                isModeless={false}
+                isDarkOverlay={false}
```

Comment:
We use dark overlay in other modals, so I think this one should be dark as well.